services/src/test/java/io/scalecube/services/ClusterSenderTest.java

```diff
@@ -0,0 +1,67 @@
+package io.scalecube.services;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertTrue;
+
+import io.scalecube.cluster.Cluster;
+
+import org.junit.Test;
+
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+public class ClusterSenderTest {
+
+  @Test
+  public void test_cluster_sender() throws InterruptedException {
+
+    Cluster cluster = Cluster.joinAwait();
+    ClusterSender sender = new ClusterSender(cluster);
+
+    assertEquals(cluster.address(), sender.address());
+
+    CountDownLatch latch = new CountDownLatch(1);
+    sender.listen().subscribe(onNext -> {
+      latch.countDown();
+    });
+
+    latch.await(1, TimeUnit.SECONDS);
```

Comment:
All the comments which I wrote for `TransportSenderTest` is applicable in same way for this class as well